app/api/resources/mentorship_relation.py

```diff
@@ -335,3 +336,137 @@ def put(cls, request_id, task_id):
+
+        if 'comment' not in data:
+            return messages.COMMENT_FIELD_IS_MISSING
+
+        return {}
+
+@mentorship_relation_ns.route('mentorship_relation/<int:request_id>/task/<int:task_id>/comments')
+class ReadComments(Resource):
+
+    @classmethod
+    @jwt_required
+    @mentorship_relation_ns.doc('create_comment_for_task')
+    @mentorship_relation_ns.expect(auth_header_parser)
+    @mentorship_relation_ns.response(
+        200, 'Return all comments for a given task',
+        model=list_comment_response_body
+        )
+    @mentorship_relation_ns.response(400, '%s'%messages.COMMENT_FIELD_IS_MISSING)
+    @mentorship_relation_ns.response(401, '%s'%messages.USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION)
+    @mentorship_relation_ns.response(404, '%s\n%s'%(
+        messages.TASK_DOES_NOT_EXIST, messages.MENTORSHIP_RELATION_DOES_NOT_EXIST
+        ))
+    def get(cls, request_id, task_id):
+        """
+        Read comments for a task.
+        """
+
+        user_id = get_jwt_identity()
+        response = CommentDAO.get_task_comments(user_id, request_id, task_id)
+
+        return response
+
+@mentorship_relation_ns.response(400, '%s'%messages.COMMENT_FIELD_IS_MISSING)
+@mentorship_relation_ns.response(401, '%s'%messages.USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION)
+@mentorship_relation_ns.response(404, '%s\n%s\n%s'%(
+    messages.TASK_DOES_NOT_EXIST, messages.MENTORSHIP_RELATION_DOES_NOT_EXIST,
+    messages.COMMENT_DOES_NOT_EXIST
+    ))
+@mentorship_relation_ns.route('mentorship_relation/<int:request_id>/task/<int:task_id>/comment/<int:comment_id>')
+class Comment(Resource):
+
+    @classmethod
+    @jwt_required
+    @mentorship_relation_ns.doc('read_comment_for_task')
+    @mentorship_relation_ns.expect(auth_header_parser)
+    @mentorship_relation_ns.response(200,"Comment successfully found", list_comment_response_body)
+    def get(cls, request_id, task_id, comment_id):
+        """
+        Read comment for a task.
+        """
+
+        user_id = get_jwt_identity()
+        response = CommentDAO.get_task_comment(user_id, request_id, task_id, comment_id)
+
+        return response
+
+    @classmethod
+    @jwt_required
+    @mentorship_relation_ns.doc('delete_comment_for_task')
+    @mentorship_relation_ns.expect(auth_header_parser)
+    @mentorship_relation_ns.response(200,"%s"%messages.COMMENT_WAS_DELETED_SUCCESSFULLY)
+    def delete(cls, request_id, task_id, comment_id):
+        """
+        Delete comment for a task.
+        """
+
+        user_id = get_jwt_identity()
+        response = CommentDAO.delete_task_comment(user_id, request_id, task_id, comment_id)
+
+        return response
+
+    @classmethod
+    @jwt_required
+    @mentorship_relation_ns.doc('update_comment_for_task')
+    @mentorship_relation_ns.expect(auth_header_parser, create_comment_request_body)
+    @mentorship_relation_ns.response(200,"%s"%messages.UPDATED_COMMENT_WITH_SUCCESS)
+    def put(cls, request_id, task_id, comment_id):
```

Comment:
this api has more possible responses which are not documented here. Can you please add them here? Such as 404 for task and comment not found. Can you check for other APIs you created, which might be missing that.

Reply:
Same as above
![image](https://user-images.githubusercontent.com/50169911/72588979-ed866a00-391f-11ea-85e4-bd0ab1595840.png)